service/history/api/consistency_checker.go

```diff
@@ -37,6 +37,7 @@ import (
 	"go.temporal.io/server/common/namespace"
 	"go.temporal.io/server/common/persistence"
 	"go.temporal.io/server/common/persistence/versionhistory"
+	historyCache "go.temporal.io/server/service/history/cache"
```

Comment:
I think `hcache` is more canonical go import prefix for `history/cache` package.

Reply:
moving and renaming to `wcache`.